llama_index/indices/service_context.py

```diff
@@ -72,7 +72,7 @@ def from_defaults(
         node_parser: Optional[NodeParser] = None,
         llama_logger: Optional[LlamaLogger] = None,
         callback_manager: Optional[CallbackManager] = None,
-        chunk_size_limit: Optional[int] = None,
```

Comment:
why did we change this variable name? this is gonna break BW compat again

Reply:
We can keep and mark as deprecated.

This is just a first pass to show the intended change.